addons/hr_expense/report/hr_expense_report.xml

```diff
@@ -49,7 +49,8 @@
                                     <td>
                                         <span t-field="line.name"/>
                                         <span t-field="line.description"/><br/>
-                                        <span t-field="line.analytic_account_id.name"/>
+                                        <!-- TODO make a print version
+                                        <span t-field="line.analytic_distribution" widget="analytic_distribution"/> !-->
```

Comment:
It can be easy to just create a function that returns a list of `(name, percentage)` and iterate on it here in a small table